### PR TITLE
Replace tasks help message with screenshot and add task to automate the screenshot process

### LINKED ISSUE #97: GitHub is not able to render myst based content in documentation
**Describe the bug**

Github is not able to render myst based documentation
image

**Version**
* OS:  all

**To Reproduce**

Go to https://github.com/astronomer/ask-astro/blob/main/docs/local_development.md

**Expected behavior**

Render the text

**Screenshots**

image

**Additional context**

### PR DESCRIPTION
**Pull request by Lee-W**
* docs: replace task help message text with screenshots
* build(tasks): add docs.generate-tasks-help-screenshot task
* build(poetry): add rich for group doc

close #97

### CODE REVIEW

**Comment by Lee-W** [comment]
image

File: docs/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<br />
+local_development.md<br />
```
Comment (sunank200) [inline docs/README.md:1]:
```suggestion
<br />
```

**Review by sunank200 — APPROVED** [review]

File: docs/README.md (hunk 1 above)
Comment (pankajastro) [inline docs/README.md:1]: not sure if it intentional but looking weird I have tried to remove it https://github.com/astronomer/ask-astro/pull/121

Comment (sunank200) [reply docs/README.md:1]: Thank you

**Pull request merged**